yal/assessments.py

```diff
@@ -150,3 +154,42 @@ async def state_survey_process_answer(self):
                 self.save_metadata("assessment_score", score)
 
         return await self.go_to_state("state_survey_question")
+
+    async def state_assessment_later_submit(self):
+        msisdn = normalise_phonenumber(self.inbound.from_addr)
+        whatsapp_id = msisdn.lstrip(" + ")
+
+        reminder_time = get_current_datetime() + timedelta(hours=23)
+        self.save_metadata("assessment_reminder", reminder_time.isoformat())
```

Comment:
The `update_profile` will also update this metadata, so we don't need to worry about doing that here.

Reply:
Oh! don't we do that for other things? I copy pasta'd from another flow :)

Reply:
Ah, I must've missed one then. Yeah I combined it so that the metadata and the rapidpro contact wouldn't be able to go out of sync.